### ISSUE TITLE
:penguin: Add ARM/RPI support for Ubuntu 20.04 and 22.04 images

### ISSUE THREAD

**Issue by venkatnsrinivasan** [kairos-io/kairos#645]
**Is your feature request related to a problem? Please describe.**
Currently kairos rpi images are available for alpine and opensuse. Would like ubuntu images to be added as well.

**Describe the solution you'd like**
ubuntu  20.04 and 22.04 rpi images for rpi 3 and 4

**Comment by c0ffee** [comment]
anyone is working on this?

**Comment by mudler** [comment]
not currently, cards in progress are tracked in our sprint board: https://github.com/orgs/kairos-io/projects/1/views/1

**Comment by sdwilsh** [comment]
What else is involved with this other than these steps?
1) New docker files (`images/Dockerfile.ubuntu-20-lts-arm-rpi`, `images/Dockerfile.ubuntu-22-lts-arm-rpi`)
2) Populate the flavors in these files: https://github.com/kairos-io/kairos/search?q=rpi&type=code
3) Testing the image (can maybe be done with images built by CI on the PR?)

**Comment by c0ffee** [comment]
Yeah I would assume it is mainly this, figuring out what packages are needed in the dockerfiles

**Comment by sdwilsh** [comment]
In my experience working with Ubuntu on an rpi, generally I just need to install `linux-modules-extra-raspi` to get the same set of stuff I have in a non-rpi host.

**Comment by mauromorales** [comment]
I just realized that I'm missing the standard images

**Comment by mauromorales** [comment]
There are a few issues with the releases but they are being tracked via different tickets. Other than that I think we can mark this as done
